tutorials/scripting/gdscript/gdscript_advanced.rst

```diff
@@ -322,23 +322,22 @@ Iterating using the C-style for loop in C-derived languages can be quite complex
 
 .. code-block:: cpp
 
-    const char* strings = new const char*[50];
+    const char* stringA = new char[50];
+    const std::string stringsB = new char[50];
```

Comment:
```suggestion
    const std::string stringB = new char[50];
```

Also this will cause a memory leak, so better to not give an example like this, but do
```suggestion
    const std::string stringB = std::string(50, ' ');
```
Or something